src/component/variantPage/BasicInfo.tsx

```diff
@@ -340,7 +340,9 @@ export default class BasicInfo extends React.Component<IBasicInfoProps> {
                     to={
                         '/annotation/' +
                         this.props.variant +
-                        '?fields=hotspots%2Cmutation_assessor%2Cmy_variant_info%2Cptms%2Cannotation_summary'
+                        // remove mutation assessor for now
+                        // '?fields=hotspots%2Cmutation_assessor%2Cmy_variant_info%2Cptms%2Cannotation_summary'
+                        '?fields=hotspots%2Cmy_variant_info%2Cptms%2Cannotation_summary'
```

Comment:
@leexgh it might make sense to set SHOW_MUTATION_ASSESSOR variable at a higher level, so you can re-use it here.